build/test-aot.ps1

```diff
@@ -0,0 +1,50 @@
+param([string]$targetNetFramework)
+
+$projectName='Microsoft.Identity.Web.AotCompatibiliy.TestApp'
+$rootDirectory = Split-Path $PSScriptRoot -Parent
+$publishOutput = dotnet publish $rootDirectory/tests/$projectName/$projectName.csproj --self-contained -nodeReuse:false /p:UseSharedCompilation=false
```

Comment:
```suggestion
$publishOutput = dotnet publish $rootDirectory/tests/$projectName/$projectName.csproj -nodeReuse:false /p:UseSharedCompilation=false
```

`--self-contained` shouldn't be necessary.

Reply:
I have removed it.